### PR TITLE
doc changes for 5.0

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: docs/source/intro.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ We have various example scripts and notebooks for using ipyparallel in our
 Some of these are covered in more detail in the :ref:`examples
 <parallel_examples>` section.
 
-.. __: http://nbviewer.ipython.org/github/ipython/ipython/blob/master/examples/Parallel%20Computing/Index.ipynb
+.. __: http://nbviewer.ipython.org/github/ipython/ipyparallel/blob/master/examples/Index.ipynb
```
Comment (Carreau) [inline docs/source/intro.rst]: ipython.org -> jupyter.org ?

File: docs/source/intro.rst [hunk 2]
```diff
@@ -260,12 +256,12 @@ everything is working correctly, try the following commands:
 
 .. sourcecode:: ipython
 
-	In [1]: from ipyparallel import Client
+	In [1]: import ipyparallel as ipp
 
-	In [2]: c = Client()
+	In [2]: c = ippClient()
```
Comment (Carreau) [inline docs/source/intro.rst]: `ipp<dot>Client` (missing dot)

**Comment by Carreau** [comment]
2 minors typos.

**Comment by minrk** [comment]
Typos fixed.

**Pull request merged**